tests/secret_to_ckks/invalid.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: heir-opt --split-input-file --secret-to-ckks="poly-mod-degree=1024" --verify-diagnostics %s | FileCheck %s
+
+// Tests invalid secret types
+
+// expected-error@below {{expected batched secret types to be tensors with dimension matching ring parameter}}
+module {
+  func.func @test_invalid_dimension(%arg0 : !secret.secret<tensor<1000xi1>>) -> (!secret.secret<tensor<1000xi1>>) {
+    return %arg0 : !secret.secret<tensor<1000xi1>>
+  }
+}
+
+// -----
+
+// CHECK: test_valid_dimension
+func.func @test_valid_dimension(%arg0 : !secret.secret<tensor<1024xi1>>) -> (!secret.secret<tensor<1024xi1>>) {
+  return %arg0 : !secret.secret<tensor<1024xi1>>
```

Comment:
Related to the replication: if we did add that in, then we would then add a test here to verify that the type conversion resolved properly.